packages/react-native-bpk-component-calendar/react-native-bpk-component-calendar.podspec

```diff
@@ -0,0 +1,19 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = "react-native-bpk-component-calendar"
+  s.version      = package['version']
+  s.summary      = "Backpack calendar component for React Native"
+
+  s.authors      = { "backpack" => "backpacksquad@skyscanner.net" }
+  s.homepage     = "https://backpack.github.io/components/calendar"
+  s.license      = "MIT"
+  s.platform     = :ios, "9.0"
+
+  s.source       = { :git => "https://github.com/Skyscanner/backpack-react-native.git" }
+  s.source_files  = "src/ios/CalendarBridge/**/*.{h,m}"
+
+  s.dependency 'React'
```

Comment:
Do we need this dep here. By definition, the RN codebase to which this library is added has React already.

Reply:
Let's just follow the lead of what other components that include bridges do, I guess.